docs/Sources/tuist/tuist.docc/Articles/Tuist/Users/Tutorials/Tuist Tutorial/Getting-Started/Add External Dependencies/external-dependencies.tutorial

```diff
@@ -7,15 +7,15 @@
 
     @Section(title: "Defining package dependencies") {
         @ContentAndMedia {
-            External dependencies are defined in the `Tuist/Package.swift` manifest file.
+            External dependencies are defined in the `Package.swift` manifest file.
```

Comment:
Where should this `Package.swift` be located? I think we need to be more opinionated about this and make sure that our docs reflect that.